concent_api/api-integration-additional-verification-test.py

```diff
@@ -151,7 +151,7 @@ def test_case_1_test_for_positive_case(cluster_consts, cluster_url, test_id):
         source_file_size_2
     ))
 
-    time.sleep(10)
+    time.sleep(15)
```

Comment:
Ok, but consider instead of hard coded 15 import as variable from settings. If settings will change i don't think we will remember this line :)

Reply:
Yeah. We should be using times based on settings in situations like this. There was even a task for refactoring all the code to do this so let's not go back to the state from before.